solutions/standard/variables.tf

```diff
@@ -352,19 +352,25 @@ variable "admin_pass_sm_secret_name" {
 ##############################################################
 
 variable "existing_code_engine_project_id" {
-  description = "Existing code engine project ID to deploy Kibana. If no value is passed, a new code engine project will be created."
   type        = string
+  description = "Existing code engine project ID to deploy Kibana. If no value is passed, a new code engine project ID will be created."
   default     = null
 }
 
 variable "enable_kibana_dashboard" {
   type        = bool
-  description = "Set it true to deploy Kibana in code engine. NOTE: Kibana image is coming direcly from the official registry (https://www.docker.elastic.co/) and not certified by the IBM."
+  description = "Set to true to deploy Kibana in Code Engine. NOTE: By default, the Kibana image will be pulled from the official Elastic registry (docker.elastic.co) and is not certified by IBM, however this can be overridden using the `kibana_registry_namespace_image` and `kibana_image_digest` input."
   default     = false
 }
 
-variable "elasticsearch_full_version" {
-  description = "(Optional) Full version of the Elasticsearch instance in the format `x.x.x` to deploy Kibana dashboard. If no value is passed, data lookup will fetch the full version using the Elasticsearch API, see https://github.com/elastic/kibana?tab=readme-ov-file#version-compatibility-with-elasticsearch"
+variable "kibana_registry_namespace_image" {
   type        = string
+  description = "The full Elasticsearch version (format `x.x.x`) required to deploy the Kibana dashboard. This value is used only when `enable_kibana_dashboard` is set to true. By default, the image is pulled from `docker.elastic.co/kibana/kibana`. If overridden, ensure the version is compatible with the Elasticsearch instance."
```

Comment:
The description here is out of date and does not match the variable purpose. The variable now takes in the image reference in the format of `[registry-url]/[namespace]/[image]`